### PR TITLE
aws_kms: Support setting PendingWindowInDays (Deletion Delay) and fix tests

### PR DESCRIPTION
**Pull request by tremble**
##### SUMMARY

Adds support for setting `pending_window` (the number of days a CMK will sit waiting to be deleted.
Fixes failures when we don't have the GetKeyRotationStatus permission
Fixes integration tests and marks them supported

##### ISSUE TYPE

- Bugfix Pull Request
- Feature Pull Request

##### COMPONENT NAME

aws_kms

##### ADDITIONAL INFORMATION

Writing tests for #199 highlighted that aws_kms also handled missing 'Get' permissions poorly, follow up and handle them better.

Pending mattclay/aws-terminator#106 for CI permissions

### CODE REVIEW

**Comment by tremble** [comment]
@jillr I've got this one running under CI now.  Some minor bugs needed fixing.

There's some instability in the tests, which seem to be related to doing things like re-updating policies quickly after an update.  I think the value of getting these tests in place outweighs waiting until we've ironed all of the flakes out, so I've marked the tests 'unstable' so they're at least run when someone submits a KMS change.

**Review by jillr — COMMENTED** [review]
All uses of version_added should now be 1.4.0, otherwise lgtm

File: plugins/modules/aws_kms.py [hunk 1]
```diff
@@ -117,6 +117,15 @@
...
     type: dict
+  pending_window:
+    description:
+    - The number of days between requesting deletion of the CMK and when it will actually be deleted.
+    - Only used when I(state=absent) and the CMK has not yet been deleted.
+    - Valid values are between 7 and 30 (inclusive).
+    - 'See also: U(https://docs.aws.amazon.com/kms/latest/APIReference/API_ScheduleKeyDeletion.html#KMS-ScheduleKeyDeletion-request-PendingWindowInDays)'
+    type: int
+    aliases: ['deletion_delay']
+    version_added: 1.5.0
```
Comment (tremble) [inline plugins/modules/aws_kms.py:128]:
```suggestion
    version_added: 1.4.0
```

File: plugins/modules/aws_kms_info.py [hunk 2]
```diff
@@ -16,12 +16,31 @@
...
 options:
+  alias:
+    description:
+      - Alias for key.
+      - Mutually exclusive with I(key_id) and I(filters).
+    required: false
+    aliases:
+      - key_alias
+    type: str
+    version_added: 1.3.0
```
Comment (tremble) [inline plugins/modules/aws_kms_info.py:27]:
```suggestion
    version_added: 1.4.0
```

File: plugins/modules/aws_kms_info.py [hunk 3]
```diff
@@ -16,12 +16,31 @@
...
+  key_id:
+    description:
+      - Key ID or ARN of the key.
+      - Mutually exclusive with I(alias) and I(filters).
+    required: false
+    aliases:
+      - key_arn
+    type: str
+    version_added: 1.3.0
```
Comment (tremble) [inline plugins/modules/aws_kms_info.py:36]:
```suggestion
    version_added: 1.4.0
```

**Review by jillr — APPROVED** [review]

**Pull request merged**